### ISSUE TITLE
Install instructions?

### ISSUE THREAD

**Issue by blopker** [mja129/htmldjango-zed#1]
Hey! Thanks for putting this together, but I'm not sure how to install it in Zed. I'm not seeing it in the installation extensions screen? Any tips here? Thank you!

**Comment by mja129** [comment]
Hey!
Yea, this whole extension is a little too janky thus far for me to be comfortable sticking it in Zed's official extension repo, there's a few kinks I'd like to iron out, i.e inline javascript.
If you'd like to give it a shot, you can clone this repo and then add the folder as a dev extension, using the button in the very top right of the 'Extensions' page in Zed.
If you do end up giving it a try, and you have any huge gripes with it (very likely, I think) please let me know :)

**Comment by blopker** [comment]
Fair enough! It seems like the whole Zed ecosystem is just starting to get legs, so all that makes sense. I'm still exploring and figuring out what is still missing in Zed from VSCode. I'll let you know if I have any feedback though.

**Comment by mja129** [comment]
Great!
I agree, and I feel like Zed could be really amazing once the possibilities for extensions expand beyond parsers and language servers.
